workflows/phylogenetics/wf_augur.wdl

```diff
@@ -167,14 +168,14 @@ workflow augur {
   }
```

Comment:
please update lines 164 and 148 with the new variable `build_name_updated`

Reply:
Done! Good catch!